docs/includes/apiargs-MongoDBCollection-method-updateOne-option.yaml

```diff
@@ -2,6 +2,18 @@ source:
   file: apiargs-MongoDBCollection-common-option.yaml
   ref: upsert
 ---
+arg_name: option
```

Comment:
Can we move this definition to `apiargs-MongoDBCollection-common-option.yaml`, like `upsert`?